Source/Samples/Sections/Widgets/EntrySection.cs

```diff
@@ -0,0 +1,81 @@
+﻿using Gtk;
+
+namespace Samples
+{
+    [Section(Name= "Entry", Category = Category.Widgets)]
+    class EntrySection : ListSection
+    {
+        public EntrySection()
+        {
+            AddItem(CreateSimpleEntry());
+            AddItem(CreateSimpleCenterAlignedTextEntry());
```

Comment:
No point in having 2 samples show the same property, remove either center or right aligned one.